starter/src/main/java/org/teiid/spring/autoconfigure/TeiidServer.java

```diff
@@ -203,13 +204,14 @@ public void addDataSource(VDBMetaData vdb, String sourceBeanName, Object source,
         }
     }
 
-    String addOverrideTranslator(VDBTranslatorMetaData translator, ExternalSource es, ApplicationContext context) {
+    String addOverrideTranslator(String overrideName, Map<String, String> properties, ExternalSource base,
```

Comment:
@shawkins  Changes below to fix the issue you raised with translators

Reply:
That's nearly there, but there's no guarantee that the source name won't conflict with an existing translator name.

create server something foreign data wrapper file;
create server file foreign data wrapper excel;
...

then set a property for excel